plugins/wp/index.js

```diff
@@ -40,8 +39,8 @@ module.exports = function (app) {
         modified: moment(post.modified).format('Do MMMM YYYY'),
         image: post.featured_image,
         categories: post.categories ? Object.keys(post.categories) : [],
-        caption: attachments_caption.join(''),
-        alt: attachments_alt.join('')
+        caption: featuredImageAttachments.caption,
```

Comment:
@mpolidori 

please make them not just `caption` and `alt` but `imageCaption` and `imageAlt`, because the post itself can have a caption probably.

Reply:
Just saw this. Yeah, this makes sense then.